e2e/test/GlobalSuppressions.cs

```diff
@@ -0,0 +1,9 @@
+﻿// This file is used by Code Analysis to maintain SuppressMessage
+// attributes that are applied to this project.
+// Project-level suppressions either have no target or are given
+// a specific target and scoped to a namespace, type, member, etc.
+
+using System.Diagnostics.CodeAnalysis;
+
+[assembly: SuppressMessage("Style", "IDE1006:Naming Styles", Justification = "Missing Async suffix on test method names. To maintain VSO history we will ignore these naming issues.", Scope = "module")]
```

Comment:
I'd say we don't append Async to test names because our test names are about what they test and the keyword Async could muddy what that means.

Sure, also for historic reasons.

FWIW, the service team had the same conclusion for Project Unicorn.

Reply:
Sure, I will update the justification